packages/peregrine/src/Router/resolveUnknownRoute.js

```diff
@@ -23,7 +21,34 @@ export default function resolveUnknownRoute(opts) {
             id: res.id,
             matched: true
         }));
-    });
+    }
+
+    if (!preloadDone) {
+        const preloaded = document.getElementById('url-resolver');
+        if (preloaded) {
+            const rejectPreload = e =>
+                console.error(
+                    'Unable to read preload!',
+                    preloaded.textContent,
+                    e
+                );
+            try {
+                return handleResolverResponse(
+                    JSON.parse(preloaded.textContent)
+                ).then(x => {
+                    preloadDone = true;
+                    return x;
+                }, rejectPreload);
+            } catch (e) {
+                rejectPreload();
```

Comment:
It looks like `rejectPreload` accepts a single argument, `e`. It's not being called with any arguments here, though. Did you mean to pass `e` to it?